app-connector-client/server/routes/connector.js

```diff
@@ -161,7 +161,7 @@ async function connect(req, res) {
         var crt = await callCSRUrl(tokenResponse.csrUrl, csr, insecure)
         var infoResponse = await callInfoUrl(tokenResponse.api.infoUrl, crt, connection.privateKey(), insecure)
 
-        var domains = new url.URL(infoResponse.urls.metadataUrl).hostname.split(".")
+        var domains = new url.URL(infoResponse.urls.metadataUrl).hostname.replace("gateway.", "");
```

Comment:
Have you checked it with minikube scenario?

Reply:
yes